### PR TITLE
Sensor tracker

### PR DESCRIPTION
**Pull request by havardh**

### CODE REVIEW

File: projects/tracker/ram_store.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+static mut STORE: [[u8; 512]; 8] = [[0; 512]; 8];
+static mut INDEX: usize = 0;
+
+pub fn init() {}
```
Comment (sondrele) [inline projects/tracker/ram_store.rs:4]: Unnecessary function?

File: projects/tracker/SensorTracker.rs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        }
+    }
+}
+
+#[no_mangle]
+#[allow(non_snake_case)]
+pub extern fn RTC_IRQHandler() {
+
+    rtc::int_clear(rtc::RTC_IEN_COMP0);
+    temperature::on_rtc();
```
Comment (sondrele) [inline projects/tracker/SensorTracker.rs:66]: Suggestion: Is the temperature module the only thing using RTC? If so, it is possible can replace the on_rtc() with the actual RTC_IRQHandler.

File: src/modules/dma.rs [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+        self.device.configure_channel(&dma::CfgChannel {
+            high_pri: true,
+            enable_int: true,
+            select: on.get(),
+            cb: unsafe {
+                transmute(match self.device.channel {
+                    0 => &DMA0_CB,
+                    1 => &DMA1_CB,
+                    _ => panic!()
```
Comment (sondrele) [inline src/modules/dma.rs]: Maybe write a message, in case this ever happens?

File: src/modules/dma.rs [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    pub fn refresh(&mut self) -> &mut Dma {
+
+        let null: *mut c_void = unsafe { transmute(ptr::null::<u8>()) };
+        self.device.activate_basic::<u8>(true, false, null, null, self.size() - 1);
+
+        self
+    }
+
+    fn size(&self) -> u32 {
+        512
```
Comment (sondrele) [inline src/modules/dma.rs]: Safe to assume that this will always be the case for our current usage?

File: projects/tracker/SensorTracker.rs (hunk 2 above)
Comment (havardh) [inline projects/tracker/SensorTracker.rs:66]: Doesn't work, and i will add more to this handler later.

File: projects/tracker/ram_store.rs (hunk 1 above)
Comment (havardh) [inline projects/tracker/ram_store.rs:4]: Will be used for flash implementation.

File: projects/tracker/temperature.rs [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    let data = adc.data_single_get();
+
+    unsafe {
+        if BUFFER.push(data as u8) {
+            store::write(&BUFFER.data);
+        }
+    }
+}
+
+fn cb(dma: &mut dma::Dma) {
```
Comment (sondrele) [inline projects/tracker/temperature.rs:125]: Give a more descriptive name

**Pull request merged**